guidelines/1.3.2.md

```diff
@@ -1,32 +1,67 @@
-#### [1.3.2 Meaningful sequence (A)](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-sequence.html)
+# Order of elements in code
 
-When the order of content is important, the content order must be preserved no matter how it is presented to the user. This ensures that the order of content is meaningful whether someone is looking at it, listening to it with a screen reader, or has stripped out visual styling using a browser plugin.
+> Make sure that elements appear in a logical reading order in code, so that they are presented in a meaningful order to screen reader users.
 
-##### Requirements / What to do?
+## What?
 
-*   The visual order of content matches the underlying code order;
-*   Tabular data is presented using proper HTML markup (`<table>`, `<tr>`, `<th>`, and `<td>` elements).
+When the order of content is important, the content order must be preserved no matter how it is presented to the user.
 
-##### Common mistakes
+### Requirements
 
-*   The DOM order does not match the visual order because CSS properties like flexbox and grid-layout have been used to alter the visual presentation;
-*   When CSS styles are disabled, the content order is meaningless;
-*   Tabular data is displayed using CSS to present the appearance of a table, but the proper HTML markup has not been used;
-*   The tabindex attribute has been used to force a tab order through the content.
+* The visual order of content matches the underlying code order;
+* Tabular data is presented using proper HTML markup (`<table>`, `<tr>`, `<th>`, and `<td>` elements).
+
+### Official guidelines
+
+* [1.3.2 "Meaningful sequence" (A) in the Web Content Accessibility Guidelines](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-sequence.html)
+
+## Why?
+
+This ensures that the order of content is meaningful whether someone is looking at it, listening to it with a screen reader, or has stripped out visual styling using a browser plugin.
+
+## How?
+
+### Design
+
+Information on how designers can fulfil this requirement.
+
+#### Supporting docs and references
+
+* Links to relevant docs and references for designers.
 
-##### Code examples
+---
 
-###### iOS
+### iOS
+
+Setting a custom order for child elements:
 
 ```swift
-// Setting a custom order for child elements
 containerView.isAccessibilityElement = false
 containerView.accessibilityElements = [firstLabel, secondLabel, button]
 ```
 
+#### Supporting docs and references
+
 * [UIAccessibilityContainer developer reference on apple.com](https://developer.apple.com/documentation/uikit/accessibility/uiaccessibilitycontainer "developer.apple.com reference")
 
-##### Useful resources
+---
+
+### Web
+
+#### Common mistakes
```

Comment:
I left this in here in 1.3.2, but I took it out of 1.3.1, which I now think I'll add back in before merging. Unless you think it adds noise, in which case I'll take it out of here as well. I'm okay with there being common mistakes, as long as they're really adding value and not just extra content that makes it more daunting.